.github/workflows/rust.yml

```diff
@@ -12,11 +12,13 @@ env:
 jobs:
   build:
 
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
+    matrix:
+      os: [ ubuntu-latest, macos-latest, windows-latest ]
```

Comment:
malachite-bigint itself doesn't have any platform specific code.
multi-platform matrix will be tested from malachite project.